Dockerfile

```diff
@@ -18,6 +18,9 @@ ARG COMPOSE_VERSION=v2.24.5
 ARG SYSTEMD="false"
 ARG DOCKER_STATIC=1
 
+ARG PAGER
+ARG GIT_PAGER
+
```

Comment:
Looks like we're not using these (we only set them at runtime through `-e`)

Reply:
Oh right, we actually don't need these at build time.